stages/infra-setup/TCID-DIR-INSTALL-ON-LOCAL-HP

```diff
@@ -41,22 +43,51 @@ node() {
   # Create clusterrolebinding
   kubectl create clusterrolebinding kube-admin --clusterrole cluster-admin --serviceaccount=kube-system:default
 
+
+  # Deploy Kubera using either kubera-charts repo or Kubera official charts
+
+if [ "$USE_KUBERA_REPO" == "true" ]
+then
+
+  # Deploy Kubera from kubera-charts repo
+  echo -e "\nDeploying Kubera using kubera-charts repo\n"
+  echo -e "\nCloning $KUBERA_BRANCH branch of kubera-charts repo\n"
+  git clone -b $KUBERA_BRANCH https://$GITHUB_USERNAME:$GITHUB_PASSWORD@github.com/mayadata-io/kubera-charts.git
+  cd kubera-charts
+
+  echo -e "\nUpdating values in kubera-charts/values.yaml\n"
+  # Update release version
+  sed 's|release: .*|release: '$RELEASE'|' -i ./values.yaml
+  # Update URL for accessing Kubera
+  sed 's|url: .*|url: '$DOP_URL'|' -i ./values.yaml
+  # Update max member count in a project to 100
+  sed 's|maxMemberCountInOneProject: .*|maxMemberCountInOneProject: 100|' -i ./values.yaml
+  # Enable openebs RC builds deployment
+  sed 's|openebsRCEnable: .*|openebsRCEnable: true|' -i ./values.yaml
+  # Disable nginx-ingress deployment as it is predeployed in Rancher
+  ## Sed command to replace the third line next to "nginx-ingress:"
+  sed '/nginx-ingress:/!b;n;n;n;c\  enabled: false' -i ./values.yaml
```

Comment:
Remove these lines as nginx is not pre-deployed in Konvoy.

Reply:
Set `type.installOpenebs=false` in the values.yaml here as openebs is predeployed in cluster1 in Konvoy.